docs/fastdds/discovery/simple.rst

```diff
@@ -119,7 +119,7 @@ Initial peers
 According to the `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_ (Section 9.6.1.1), each
 |RTPSParticipant-api|
 must listen for incoming Participant Discovery Protocol (PDP) discovery metatraffic in two different ports, one linked
-with a multicast address, and another one linked to a unicast address.
+to a multicast address and another one linked to a unicast address.
```

Comment:
```suggestion
to a multicast address, and another one linked to a unicast address.
```
We normally use a comma before an "and" to splits two independent clauses.

Reply:
I believe the clauses aren't independent. The sentence is clarifying the two different types of ports, so both need to stay together. The sentence "another one linked to a unicast address" could not stand alone, and therefore isn't independent.